src/systems/userinput/devices/daydream-controller.js

```diff
@@ -10,6 +13,15 @@ export class DaydreamControllerDevice {
     this.rayObjectRotation = new THREE.Quaternion();
     this.selector = `#player-${gamepad.hand}-controller`;
     this.pose = new Pose();
+    this.sittingToStandingMatrix = new THREE.Matrix4().makeTranslation(0, 1.6, 0);
+    navigator.getVRDisplays().then(d => {
```

Comment:
Since we do this in all the controllers, might pull this out into a function that resolves to a Promise to a Matrix4